tensorflow/contrib/learn/python/learn/datasets/__init__.py

```diff
@@ -13,7 +13,9 @@
 # limitations under the License.
 # ==============================================================================
 
-"""Module includes reference datasets and utilities to load datasets."""
+"""Module includes reference datasets and utilities to load datasets. It also 
```

Comment:
first line of comments in modules (and functions) should be one line, you rephrase to something like:
`Dataset utilities and synthetic/reference datasets.`